langchain4j/src/main/java/dev/langchain4j/service/AiServices.java

```diff
@@ -171,6 +175,21 @@ public static <T> AiServices<T> builder(Class<T> aiService) {
         return new DefaultAiServices<>(context);
     }
 
+    public AiServices<T> messages(MessagesProvider messagesProvider) {
+        context.messagesProvider = messagesProvider;
+        return this;
+    }
+
+    public AiServices<T> systemMessages(Supplier<String> systemMessageSupplier) {
```

Comment:
Even if I'm not testing/using this yet I believe it is nice to allow to configure a system and/or a user message also passing a simple lambda.